colabs/mosaicml/MosaicML_Composer_and_wandb.ipynb

```diff
@@ -0,0 +1,765 @@
+{
```

Comment:
<p>Image is broken for me (in ReviewNB at least)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>it appears in the GitHub render</p><p> </p>